lib/reporters/tap.js

```diff
@@ -118,6 +119,21 @@ class TapReporter {
 		}) + os.EOL);
 	}
 
+	writeLogs(evt, flags = {error: false, title: ''}) {
+		let formattedTitle = flags.title || this.prefixTitle(evt.testFile, evt.title);
+		if (flags.error) {
+			formattedTitle = `Failed hook: ${formattedTitle}`;
+		}
+
+		this.reportStream.write(`# ${stripAnsi(formattedTitle)}${os.EOL}`);
```

Comment:
What's the reason for not using `supertap` here?

Reply:
`supertap`'s API is just having **start**, **finish** and **test**, methods, I cannot use start or finish, the only remaining one is **test** but it's injecting prefix **(not)ok**, so it's counted as test, we need to modify supertap to have another method available called hook to just write the content of the hook without outputting the test format.

Reply:
Fair enough. I suppose it'd be good to contribute a comment method to `supertap` but that's up to you.